packages/apps/reputation-oracle/server/src/modules/credentials/credential.entity.ts

```diff
@@ -0,0 +1,44 @@
+import { Column, Entity, ManyToOne } from 'typeorm';
+import { NS } from '../../common/constants';
+import { BaseEntity } from '../../database/base.entity';
+import { CredentialStatus } from '../../common/enums/credential';
+import { UserEntity } from '../user/user.entity';
+
+@Entity({ schema: NS, name: 'credentials' })
+export class CredentialEntity extends BaseEntity {
+  @Column({ type: 'varchar', unique: true })
+  public reference: string;
+
+  @Column({ type: 'text' })
+  public description: string;
+
+  @Column({ type: 'varchar' })
+  public url: string;
+
+  @Column({
+    type: 'enum',
+    enum: CredentialStatus,
+  })
+  public status: CredentialStatus;
+
+  @Column({ type: 'timestamptz' })
+  public startsAt: Date;
+
+  @Column({ type: 'timestamptz', nullable: true })
+  public expiresAt?: Date;
+}
+
+@Entity()
```

Comment:
You should specify here the options for this entity, By the way, can you check it? I ran the script but this table is not being created

Reply:
I see, as you don't specify the schema, the table is being created in public schema instead of hmt

Reply:
Added Entity configuration options : `@Entity({ schema: NS, name: 'credentials_validation' })`

 Is `credentials_validation` a correct plural name for this entity  ?

Reply:
I would say `credential_validations` as you have multiple validations in that table

Reply:
I generated the migration based on those changes and applied it.